e2e-tests/tests/sequence-editor.test.ts

```diff
@@ -0,0 +1,50 @@
+import test, { type BrowserContext, type Page } from '@playwright/test';
+import { Dictionaries } from '../fixtures/Dictionaries.js';
+import { Parcels } from '../fixtures/Parcels.js';
+import { Sequence } from '../fixtures/Sequence.js';
+let context: BrowserContext;
+let editor: Sequence;
```

Comment:
Sorry...can you also update all these `editor` references as well?

Reply:
Got it I renamed `editor` -> `sequence`